.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -290,6 +291,18 @@ jobs:
       if: inputs.tests_selector == 'cts'
       env:
         ONEAPI_DEVICE_SELECTOR: ${{ inputs.target_devices }}
+      # FIXME: For some reason the "sub_group api" test-case is failing with
+      # SIGSEGV while running test_all, so running each binary separately.
+      # BTW test_all requires a lot of resources to build it, so probably it'll
+      # be better to build each binary also separately.
+      # run: |
+      #   ./build-cts/bin/test_all --list-devices
```

Comment:
Can we have a separate step that will just "list-devices"?

Reply:
Do you mean exactly `./build-cts/bin/test_all --list-devices`, or just `sycl-ls`?

Reply:
cts's "--list-devices". I believe `sycl-ls` is part of `sycl-linux-run-tests.yml` already.

Reply:
Like this [cd94539](https://github.com/intel/llvm/pull/12811/commits/cd9453996255e0e4dc8eb0f675a13de7422a53a9)?

Reply:
Does it exit immediately after that without running tests? If so, then yes!